pegasus/sites.v3/code.org/views/curriculum_launch_skinny_banner.haml

```diff
@@ -0,0 +1,14 @@
+- if !!DCDO.get('curriculum-launch-2024', false)
+  %link{href: "/css/generated/curriculum-launch-skinny-banner.css", rel: "stylesheet"}
+
+  .curriculum-banner-skinny
+    .wrapper.flex-container.justify-space-between.align-items-center.wrap
+      .text-wrapper
+        %h5.white
```

Comment:
Can you swap this with an h2 for better accessibility?
```suggestion
        %h2.heading-sm.white
```

Reply:
Good call! I'll fix that